src/iadmin.cpp

```diff
@@ -2516,6 +2660,40 @@ usage( char *subOpt ) {
         ""
     };
 
+    char* set_pam_password_config_usage[] = {
+        "set_pam_password_config OPTION_NAME OPTION_VALUE",
+        " ",
+        ("Set the specified OPTION_NAME to OPTION_VALUE for pam_password configuration in the local zone in "
+         "R_GRID_CONFIGURATION."),
+        " ",
+        // TODO: Add descriptions for these option names...
+        "The list of accepted OPTION_NAMEs is:",
+        " - password_length",
+        " - no_extend",
+        " - password_min_time",
+        " - password_max_time",
+        ""};
+
+    char* get_pam_password_config_usage[] = {
+        "get_pam_password_config",
+        " ",
+        "Prints pam_password configuration from the local zone grid configuration as JSON.",
+        " ",
+        ("This command allows administrators to view pam_password configuration values without running queries "
+         "directly against the database."),
```

Comment:
Why parentheses here?
Wouldn't this result in the line being really long in the terminal?

Reply:
Yep. The compiler gives a warning here because if you split up the string it isn't sure if you meant to put a comma at the end of the line. I'm happy to just split this into two lines in the output if that's preferable.

Reply:
Don't we want these lines to be split via a comma so the help text honors the imaginary margin shared by all lines?

Reply:
Done